src/types/connector.ts

```diff
@@ -58,6 +58,14 @@ export interface SpotifyAggregatePayload {
         }
     }
 }
+export interface SpotifyPodcastMetadataPayload {
```

Comment:
do we ignore the other fields?

Reply:
yes, it's just stuff like the podcast title and description. Not really helpful for our use-case.